src/types/publicTypes.ts

```diff
@@ -113,93 +124,72 @@ export interface EthersProvider {
   getLogs(filter: EventFilter): Promise<RpcProviderLogEntry[]>;
 }
 
-/**
- * @deprecated Use UnclaimedDomainResponse instead (deprecated since 0.3.4)
- */
-export const UNCLAIMED_DOMAIN_RESPONSE = UnclaimedDomainResponse;
-
 export const UDApiDefaultUrl = 'https://unstoppabledomains.com/api/v1';
-export const DefaultAPI: API = {
-  url: UDApiDefaultUrl,
-};
-
-/**
- * NamingServiceSource
- * just an alias
- * @typedef {string | boolean | SourceDefinition}
- */
-export type NamingServiceSource = string | boolean | SourceDefinition;
-
 export type NamehashOptions = {
-  readonly format?: 'dec' | 'hex',
-  readonly prefix?: boolean,
+  readonly format?: 'dec' | 'hex';
+  readonly prefix?: boolean;
 };
 
-export const NamehashOptionsDefault = {format: 'hex', prefix: true} as const;
+export const NamehashOptionsDefault = { format: 'hex', prefix: true } as const;
 
 export enum DnsRecordType {
```

Comment:
Our website supports setting just 6 record types. This huge enum might give the idea that we support more. Do we actually want to specify them all here?